LayoutTests/editing/deleting/paste-with-transparent-background-color-expected.txt

```diff
@@ -13,7 +13,7 @@ After cut and paste:
 | "hello "
 | <span>
 |   class="test"
-|   style="background-color: transparent;"
+|   style="background-image: none; background-color: transparent;"
```

Comment:
Yes, that’s what we’d want the editing code to do. I don’t know how to quickly fix this right now, but I do at least know after a lot of research that the change will be to the editing code, not the CSS or style system code.

I think the core of the problem is that the editing code, for brevity, is trying to leave out various properties since they are the same as the surrounding context. But I think that for true brevity and best behavior we would want it do that only after taking shorthands into account. Right now, I think it does everything in terms of the longhands. But I honestly wasn’t able to quickly figure out _exactly_ why we end up with this, instead of just `background: transparent` or even just `background-color: transparent`.